test/ILLink.RoslynAnalyzer.Tests/RequiresAssemblyFilesAnalyzerTests.cs

```diff
@@ -352,5 +352,52 @@ public void M()
 
 			return VerifyRequiresAssemblyFilesAnalyzer (src);
 		}
+
+		[Fact]
```

Comment:
Same with the tests. Can we start to share stuff here?

Reply:
I open https://github.com/mono/linker/issues/1986 to track the refactoring work in a separate PR since I think it would have a lot of feedback.